internal/codegen/tpl_stencil.go

```diff
@@ -182,32 +181,73 @@ func (s *TplStencil) ReadFile(name string) (string, error) {
 	return string(b), nil
 }
 
+type ReadDirEntry struct {
+	Name  string
+	IsDir bool
+}
+
+// ReadDir reads the contents of a directory and returns a list of files/directories
+//
+//		{{ range $entry := stencil.ReadDir "/tests" }}
+//		  {{ if $entry.IsDir }}
+//		    {{ $entry.Name }}
+//	      {{ end }}
+//	    {{ end }}
```

Comment:
Indentation looks messed up here 🤔 (thx Go?)

I found I had to mess with tabs quite a bit to get this to work properly.